.github/workflows/npm.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: 12
+          node-version: [ '12', '14', '16' ]
```

Comment:
just realized that this should be pinned down to just one version, since we're really just using a "tool" node environment to publish our packages. kind of like `commit-watch`

Reply:
changed and squashed